Makefile

```diff
@@ -84,6 +90,10 @@ TEST_ARGS :=
 test-py: ## Clean and Make unit tests
 	python -m pytest -v csp/tests --junitxml=junit.xml $(TEST_ARGS)
 
+test-py-sanitizer: ## Clean and Make unit tests with sanitizers enabled
```

Comment:
is there a reason to not just do this by default in `coverage`? The expectation is that `make test` is quick and `make coverage` is slow/full (and run in CI)

Reply:
`coverage` is probably a mislabeling, we don't really enforce coverage (though we will soon onboard to codecov), it was really more of a "longform tests". Doesn't really matter, just might be confusing what is required to be passing for a PR to be accepted (coverage + san presumably)